src/main/java/com/team4/deskbookingappbe/controller/WelcomeController.java

```diff
@@ -0,0 +1,14 @@
+package com.team4.deskbookingappbe.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class WelcomeController {
```

Comment:
Why do you need this?

Reply:
Šiukšlė, ištrinsim, buvo testui reikalinga tada

Reply:
For testing :)